contrib/juliac-buildscript.jl

```diff
@@ -17,7 +17,7 @@ task.rngState3 = 0x3a77f7189200c20b
 task.rngState4 = 0x5502376d099035ae
 uuid_tuple = (UInt64(0), UInt64(0))
 ccall(:jl_set_module_uuid, Cvoid, (Any, NTuple{2, UInt64}), Base.__toplevel__, uuid_tuple)
-ccall(:jl_set_newly_inferred, Cvoid, (Any,), Core.Compiler.newly_inferred)
+ccall(:jl_set_newly_inferred, Cvoid, (Any,), Base.newly_inferred)
```

Comment:
This doesn't make a lot of sense to me, because I expect this array to be empty. I suspect this can be removed if you also apply the patch to make the output robust against this being `null` (https://github.com/JuliaLang/julia/pull/56128/files#diff-d1a6924143dbfd9c2f3252c060bc3036f957a7fd3339643566f6ac3e3d376f74R315). Can you try that instead?

Reply:
Sure, done. Seems to work for my usecase.